bot/exts/filters/filter_lists.py

```diff
@@ -287,10 +293,58 @@ async def deny_sync(self, ctx: Context) -> None:
         """Syncs both allowlists and denylists with the API."""
         await self._sync_data(ctx)
 
+    @command(name="filter_report")
+    async def force_send_weekly_report(self, ctx: Context) -> None:
+        """Send a list of autobans added in the last 7 days to #mod-meta."""
+        await self.send_weekly_autoban_report(ctx.channel)
+
+    @tasks.loop(time=datetime.time(hour=18))
+    async def weekly_autoban_report_task(self) -> None:
+        """Trigger autoban report to be sent if it is the desired day of the week (WEEKLY_REPORT_ISO_DAY)."""
+        if arrow.utcnow().isoweekday() != WEEKLY_REPORT_ISO_DAY:
+            return
+
+        await self.send_weekly_autoban_report()
+
+    async def send_weekly_autoban_report(self, channel: discord.abc.Messageable = None) -> None:
+        """Send a list of autobans added in the last 7 days to #mod-meta."""
+        seven_days_ago = arrow.utcnow().shift(days=-7)
+        if not channel:
+            channel = self.bot.get_channel(Channels.mod_meta)
+
+        added_autobans = defaultdict(list)
+        # Extract all autoban filters added in the past 7 days from each filter type
+        for filter_list, filters in self.bot.filter_list_cache.items():
+            filter_type, allow = filter_list.split(".")
+            allow_type = "Allow list" if allow.lower() == "true" else "Deny list"
+
+            for filter_content, filter_details in filters.items():
+                created_at = arrow.get(filter_details["created_at"])
+                updated_at = arrow.get(filter_details["updated_at"])
+                # Default to empty string so that the in check below doesn't error on None type
+                comment = filter_details["comment"] or ""
+                if max(created_at, updated_at) > seven_days_ago and "[autoban]" in comment:
+                    line = f"`{filter_content}`: {comment}"
+                    added_autobans[f"**{filter_type} {allow_type}**"].append(line)
```

Comment:
if `allow_type` is `"Allow list"`, why would we autoban based on it? This question is probably not even related to the PR, it's just that I feel like I'm missing something :sweat_smile: .

Reply:
Hah, technically autobans can be added to allow lists, which in itself seems wrong.

Reply:
True.

But you know what, we can leave this part as it is so that we can identify if an autoban is added to an allow-list.
It might be better to add a check to not allow autoban to be added to an allow-list in the first place but after a quick glance I don't think the bot will autoban based on allow list.
It is possible that a mod might accidentally add a filter to the allow list instead of blacklist and the report will help identify such cases (assuming we miss the original plain message alert sent immediately on adding an autoban filter).